istio/tests/test_check.py

```diff
@@ -442,6 +472,22 @@ def test_new_istio(aggregator, new_mesh_mixture_fixture):
     aggregator.assert_all_metrics_covered()
 
 
+def test_pilot_only_istio(aggregator, new_pilot_fixture):
+    check = Istio('istio', {}, {}, [NEW_MOCK_PILOT_ONLY_INSTANCE])
+    check.check(NEW_MOCK_PILOT_ONLY_INSTANCE)
+
+    for metric in PILOT_METRICS:
+        aggregator.assert_metric(metric)
```

Comment:
Can you add `aggregator.assert_all_metrics_covered()` here? That makes sure we covered all the metrics processed.

Reply:
Done